doc/webui/README.md

```diff
@@ -24,46 +26,233 @@ single instances can be configured independently.
 The configuration view of elektra web is similar to the tree view of the
 [qt-gui](https://github.com/ElektraInitiative/libelektra/tree/master/src/tools/qt-gui).
 
-![https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/ui_structure.png](https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/ui_structure.png)
+![https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/ui_structure.png](https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/ui_structure.png)
 
 
 ## API
 
-![https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/daemon_structure.png](https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/daemon_structure.png)
+![https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/daemon_structure.png](https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/daemon_structure.png)
+
+### Group elektrad API
 
 To access single instances, each elektra daemon (`elektrad`) provides a RESTful
 HTTP API:
 
- * **GET /version** - get `elektrad` version
- * **GET /kdb/:path** - get `path` configuration (same as `kdb get path`)
- * **POST /kdb/:path** - edit `path` configuration (same as `kdb set path`)
+#### GET /version
+
+get API and elektra version
+
++ Response 200 (application/json)
+
+        {
+            "api": 1,
+            "elektra": "0.8.17"
+        }
+
+#### /kdb/{path}
+
+access the elektra key database by specifying a `path`
+
++ Parameters
+    + path: `user/hello` (string) - path to the elektra config
+
+##### get configuration [GET]
+
+this is the same as calling `kdb get {path}`
+
++ Response 200 (application/json)
+
+        "hello world"
```

Comment:
it will be JSON in some cases, but in most cases it's probably just a string (enclosed in `"`, as per the JSON standard) or number